src/unittest/python/pmf_tests.py

```diff
@@ -9,68 +9,68 @@
 class TestPsychometricFunction(TestCase):
 
     def setUp(self):
-        self.data = np.array([[0.4, 1, 5],
-                              [0.8, 3, 5],
-                              [1.2, 5, 5],
-                              [1.3, 4, 5]])
+        self.data = np.array([[0.4, 0.2, 5],
+                              [0.8, 0.6, 5],
+                              [1.2, 1.0, 5],
+                              [1.3, 0.8, 5]])
```

Comment:
So from what i can gather, these changes affect the ```proportion correct``` parameter. I would like to know why there is a 1.0 before the 0.8 in the matrix though.  

```python
self.data = np.array([[0.4, 0.2, 5],
                                 [0.8, 0.6, 5],
                                 [1.2, 1.0, 5],
                                 [1.3, 0.8, 5]])
```

Reply:
Not sure what you're asking here. Which one? In the third row? What's wrong with that?

Reply:
In the matrix array attached to the first comment the middle column has numbers 0.2, 0.4, 1.0, 0.8 in that order. I was wondering why 0.8 comes after 1.0. Hopefully that’s more clear

Reply:
Yes! My understanding was that the numbers represented probability and not trial results. Thank you.